test/lib/ufe/testParentCmd.py

```diff
@@ -1039,6 +1039,41 @@ def testParentingToGPrim(self):
             cmds.parent("|Tree_usd|Tree_usdShape,/TreeBase/trunk",
                         "|Tree_usd|Tree_usdShape,/TreeBase/leavesXform/leaves")
 
+    def testSelectionListAfterParent(self):
+        ''' Parented object will be added to selection list. This behavior matches the native Maya parent command. '''
+
+        # create scene items for the cube and the cylinder.
+        shapeSegment = mayaUtils.createUfePathSegment(
+            "|mayaUsdProxy1|mayaUsdProxyShape1")
+        cubePath = ufe.Path(
+            [shapeSegment, usdUtils.createUfePathSegment("/cubeXform")])
+        cubeItem = ufe.Hierarchy.createItem(cubePath)
+        cylinderPath = ufe.Path(
+            [shapeSegment, usdUtils.createUfePathSegment("/cylinderXform")])
+        cylinderItem = ufe.Hierarchy.createItem(cylinderPath)
+
+        # get the USD stage
+        stage = mayaUsd.ufe.getStage(str(shapeSegment))
+
+        # global selection is empty at this point.
+        globalSelection = ufe.GlobalSelection.get()
+        self.assertEqual(len(globalSelection), 0)
+
+        # parent cube to cylinder in relative mode, using UFE path strings.
+        cmds.parent("|mayaUsdProxy1|mayaUsdProxyShape1,/cubeXform",
+                    "|mayaUsdProxy1|mayaUsdProxyShape1,/cylinderXform",
+                    relative=True)
+
+        # parented object should be in the selection list now
+        self.assertEqual(len(globalSelection), 1)
```

Comment:
Would be stronger to check what's actually in the global selection, e.g.
cubePath = ufe.Path([shapeSegment, usdUtils.createUfePathSegment("/cylinderXform/cubeXform")])
cubeItem = ufe.Hierarchy.createItem(cubePath)
self.assertEqual(globalSelection.front(), cubeItem)

Reply:
Good point. Thanks.

cc3a604fa3d934e585d899813257eecd45702a8d